src/ai/backend/manager/api/vfolder.py

```diff
@@ -2422,6 +2429,28 @@ async def clone(request: web.Request, params: Any, row: VFolderRow) -> web.Respo
     if source_proxy_name != target_proxy_name:
         raise InvalidAPIParameters("proxy name of source and target vfolders must be equal.")
 
+    async with root_ctx.db.begin_session() as sess:
+        if row["group"]:
+            log.debug("row[group]:{}", row["group"])
+            query = (
+                sa.select(GroupRow)
+                .where((GroupRow.domain_name == domain_name) & (GroupRow.id == row["group"]))
+                .options(selectinload(GroupRow.resource_policy_row))
```

Comment:
Just one little tip.
If you want to load only `max_vfolder_count` from this relationship, you can use `load_only` option.
https://github.com/lablup/backend.ai/blob/main/src/ai/backend/manager/models/session.py#L766-L768

Reply:
Thank you for helping me study ORM.
I will fix it.